Dangerfile

```diff
@@ -17,3 +17,7 @@ end
 if pr_body.length < 5
   fail "Please provide a changelog summary in the Pull Request description @#{pr_author}"
 end
+
+unless files_modified.any? { |a| a.include?("CHANGELOG.md") }
```

Comment:
YES! And if it's in a subfolder, do we want `file_modified["docs/something.md"]` or `file_modified["docs"]["something.md"]`